bootstrap/httpserver/status.go

```diff
@@ -36,33 +36,38 @@ func (s *HttpServer) getStatus(w http.ResponseWriter, r *http.Request) {
 	_, err := w.Write(data)
 	if err != nil {
 		s.logger.Info("error writing index.json response", log.Error(err))
-
 	}
-
 }
 
 func (s *HttpServer) getStatusWarningMessage() string {
 	var status string
 
-	if metricObj := s.metricRegistry.Get("BlockStorage.FileSystemIndex.LastUpdateTime"); metricObj != nil {
-		graceTimeSinceLastBlockStorageUpdate := s.config.TransactionPoolTimeBetweenEmptyBlocks() * 10
-		if graceTimeSinceLastBlockStorageUpdate < time.Minute*10 {
-			graceTimeSinceLastBlockStorageUpdate = time.Minute * 10
-		}
-		lastBlockStorageUpdateTime := s.getGaugeValueFromMetrics("BlockStorage.FileSystemIndex.LastUpdateTime")
-		if lastBlockStorageUpdateTime+int64(graceTimeSinceLastBlockStorageUpdate.Seconds()) < time.Now().Unix() {
-			status += fmt.Sprintf("Last successful blockstorage update:  %v,  was too long ago (last update includes indexing on boot) ;", lastBlockStorageUpdateTime)
+	// if no genesis exists nothing to do
+	if metricObj := s.metricRegistry.Get("Management.Data.GenesisRefTime"); metricObj != nil {
+		genesisRefTime := s.getGaugeValueFromMetrics("Management.Data.GenesisRefTime")
+		currentRefTime := s.getGaugeValueFromMetrics("Management.Data.CurrentRefTime")
+		if (genesisRefTime > 0 && currentRefTime > genesisRefTime) || genesisRefTime == 0 { // blocks should be closing
```

Comment:
what's this for ? 
 " .. || genesisRefTime == 0

Reply:
tests, gamma, and private sometimes don't really have genesis ... and if genesis exists then likely current exists :)